diff --git a/debian/rules b/debian/rules
index cf5542d..a935300 100755
--- a/debian/rules
+++ b/debian/rules
@@ -60,7 +60,7 @@ install: build
 	install -d -m 0755 "$(CURDIR)/debian/ca-certificates-udeb/etc/ssl/certs"
 	(cd mozilla; \
 	 $(MAKE) install CERTSDIR="$(CURDIR)/debian/ca-certificates-udeb/etc/ssl/certs")
-	openssl rehash -v "$(CURDIR)/debian/ca-certificates-udeb/etc/ssl/certs"
+	c_rehash -v "$(CURDIR)/debian/ca-certificates-udeb/etc/ssl/certs"
 
 # Build architecture-independent files here.
 binary-indep: build install
diff --git a/sbin/update-ca-certificates b/sbin/update-ca-certificates
index b27c6bd..17cc369 100755
--- a/sbin/update-ca-certificates
+++ b/sbin/update-ca-certificates
@@ -183,9 +183,9 @@ then
   done
   if [ "$verbose" = 0 ]
   then
-    openssl rehash . > /dev/null
+    c_rehash . > /dev/null
   else
-    openssl rehash -v .
+    c_rehash -v .
   fi
 fi
 
